### PR TITLE
v1.5.0 prep

### PR DESCRIPTION
**Pull request by Code0x58**
After #289 is merged this could be merged and tagged (maybe with a new date)

### CODE REVIEW

File: CHANGELOG.md
Comment (nathany) [inline CHANGELOG.md:3]: yeah, the date

File: .mailmap
Comment (nathany) [inline .mailmap:2]: 👍🏼

**Review by nathany — DISMISSED** [review]

**Comment by nshalman** [comment]
Is it worth trying to get #371 into shape and landed as part of v1.5.0?
Presumably it would need to be updated to include the functionality of #289?
Thoughts?

**Comment by nathany** [comment]
@nshalman Let's do FEN as a separate 1.6.0 release

File: CHANGELOG.md
Comment (nathany) [inline CHANGELOG.md:12]: Thanks for writing this up

I think it would be preferable to put the issue/PR # at the end, just to follow the format used previously (see 1.4.1, etc.)

**Comment by Code0x58** [comment]
I've updated this and set the date for Friday, but could merge it sooner or later as it will require an approval first. Once this is merged it should just be a case of tagging (hopefully I can do that).

I agree with the 1.6.0 being separate, it's probably good to release more frequently.

**Review by nathany — APPROVED** [review]

**Comment by nathany** [comment]
@Code0x58 You should be able to tag a release. Did you clone fsnotify/fsnotify or are you still working from the fork?

Or try creating a tag/release from here: https://github.com/fsnotify/fsnotify/releases

**Comment by Code0x58** [comment]
[v1.5.0](https://github.com/fsnotify/fsnotify/releases/tag/v1.5.0) is pushed and additionally marked as a release on GitHub, so any watchers will be notified.

**Comment by nathany** [comment]
Perfect. Thanks!

**Pull request merged**
